### DISCUSSION TITLE
週報用のテンプレートがほしい

### DISCUSSION THREAD

**Discussion by kagomen** [first-contributions-ja/first-contributions-ja.github.io#160] [12: 新機能の提案]
テンプレート設定しようと思ったのですが、Discussionのテンプレートがどうやらフォーム形式でしか作れないみたいです。
参考: https://docs.github.com/ja/discussions/managing-discussions-for-your-community/creating-discussion-category-forms
だったらDocsかDiscussionにテンプレート置いて、毎度コピペして使う形でも良いかなと思ったのですが、どう思いますか？

***

## 目次
- [トピックタイトル](URL)
  - 結論
 
## 次の一週間でやること
- 

***
以下フォーマット
```
## トピック: <タイトル>


## 結論


## 関連 Issues、Discussions

```

**Comment by kazzyfrog** [comment]
> テンプレート置いて、毎度コピペして使う形でも良いかなと思ったのですが、どう思いますか？

良いと思います！
Discussionsの「週報」のカテゴリに置いておく形でも良さそうですね！

**Reply by kagomen** [reply]
ありがとうございます！
ちょっと仮で作ってみます！

**Comment by kagomen** [comment]
#161  週報用テンプレート作りました。
なにか他に追記項目などありましたらテンプレートの編集をお願いします！🙏
「カテゴリー：週報」にピン留めしてあります。
